.gitignore

```diff
@@ -24,6 +24,13 @@ automappingconverter.exe
 *.lib
 *.exp
 *.intermediate.manifest
+src/libtiled/debug/
+src/tiledquick/debug/
+src/tiledquick/uic_wrapper.bat
+src/tiledquickplugin/debug/
```

Comment:
Dunno why uic_wrapper.bat is there; I just added everything that I saw that looked out of place.